api-gateway/src/main/resources/static/scripts/owner-register/owner-register.controller.js

```diff
@@ -1,46 +1,46 @@
-// 'use strict';
-//
-// angular.module('ownerRegister')
-//     .controller('OwnerRegisterController', ["$http", '$state', '$stateParams', function ($http, $state, $stateParams) {
-//         var self = this;
-//         var ownerId = $stateParams.ownerId || "";
-//         var method = $stateParams.method;
-//
-//         if (!ownerId) {
-//             self.owner = {};
-//             self.checked = false
-//         } else {
-//             $http.get("api/gateway/owners/" + ownerId).then(function (resp) {
-//                 self.owner = resp.data;
-//             });
-//             if(method == 'edit')
-//                 self.checked = false
-//             else
-//                 self.checked = true
-//         }
-//
-//         self.submitOwnerForm = function () {
-//             var id = self.owner.id;
-//             console.log(self.owner);
-//             var req;
-//             if (id){
-//                 if(method == 'edit')
-//                     req = $http.put("api/gateway/owners/" + id, self.owner);
-//                 else
-//                     req = $http.delete("api/gateway/owners/" + id, self.owner)
-//             }
-//             else
-//                 req = $http.post("api/gateway/owners", self.owner);
-//
-//             req.then(function () {
-//                 $state.go('owners');
-//             }, function (response) {
-//                 var error = response.data;
-//                 error.errors = error.errors || [];
-//                 alert(error.error + "\r\n" + error.errors.map(function (e) {
-//                     return e.field + ": " + e.defaultMessage;
-//                 }).join("\r\n"));
-//             });
-//         };
-//     }]);
-//
+'use strict';
+
+angular.module('ownerRegister')
+    .controller('OwnerRegisterController', ["$http", '$state', '$stateParams', function ($http, $state, $stateParams) {
+        var self = this;
+        var ownerId = $stateParams.ownerId || "";
+        var method = $stateParams.method;
+
+        if (!ownerId) {
+            self.owner = {};
+            self.checked = false
+        } else {
+            $http.get("api/gateway/owners/" + ownerId).then(function (resp) {
+                self.owner = resp.data;
+            });
+            if(method == 'edit')
+                self.checked = false
+            else
+                self.checked = true
+        }
+
+        self.submitOwnerForm = function () {
+            var id = self.owner.id;
+            console.log(self.owner);
+            var req;
+            if (id){
+                if(method == 'edit')
+                    req = $http.put("api/gateway/owners/" + id, self.owner);
```

Comment:
why do you still have the put and delete in register when the only thing that the register controller / page is doing is a post ?

Reply:
youssef. i did this entire page to fulfil your prerequisites you asked of me on wednesday. approve my merge and let me submit. everything works. thanks.

Reply:
Cleanup and refactoring can be done at a later point, considering the fact he was reusing elements from another aspect of the site.

Reply:
@Youssefchahboune Done.